### PR TITLE
[bitnami/contour] Update tlscertificatedelegations.projectcontour.io v0.2.9 > v0.5.0

### PR DESCRIPTION
**Pull request by VA-PLE**
Update tlscertificatedelegations.projectcontour.io

### CODE REVIEW

**Review by Mauraza — CHANGES_REQUESTED** [review]
Hi @VA-PLE,

Thanks for the PR! 
Could you update the version of the chart?

File: bitnami/contour/resources/tlscertificatedeligations.yaml [hunk 1]
```diff
@@ -20,13 +20,18 @@ spec:
   - name: v1
     schema:
       openAPIV3Schema:
-        description: TLSCertificateDelegation is an TLS Certificate Delegation CRD specification. See design/tls-certificate-delegation.md for details.
+        description: TLSCertificateDelegation is an TLS Certificate Delegation CRD
+          specification. See design/tls-certificate-delegation.md for details.
```
Comment (Mauraza) [inline bitnami/contour/resources/tlscertificatedeligations.yaml:24]: Could you leave it as it was before?

File: bitnami/contour/resources/tlscertificatedeligations.yaml [hunk 2]
```diff
@@ -20,13 +20,18 @@ spec:
...
         properties:
           apiVersion:
-            description: 'APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
```
Comment (Mauraza) [inline bitnami/contour/resources/tlscertificatedeligations.yaml:29]: same here

File: bitnami/contour/resources/tlscertificatedeligations.yaml [hunk 3]
```diff
@@ -20,13 +20,18 @@ spec:
...
             type: string
           kind:
-            description: 'Kind is a string value representing the REST resource this object represents. Servers may infer this from the endpoint the client submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
```
Comment (Mauraza) [inline bitnami/contour/resources/tlscertificatedeligations.yaml:34]: same
could you leave it as it was before in all the descriptions?

**Review by Mauraza — CHANGES_REQUESTED** [review]

**Comment by VA-PLE** [comment]
@Mauraza i used data from official contour.yaml https://raw.githubusercontent.com/projectcontour/contour/release-1.17/examples/render/contour.yaml

**Comment by Mauraza** [comment]
Hi @VA-PLE,

The length of the line is a style parameter, the content will be the same.

**Pull request closed without merging**